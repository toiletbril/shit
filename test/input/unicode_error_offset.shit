echo юникод; does_not_exist*
