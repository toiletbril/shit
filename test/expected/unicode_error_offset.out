[CompoundList]
  [CompoundListCondition, None]
    [SimpleCommand "echo юникод"]
  [CompoundListCondition, None]
    [SimpleCommand "does_not_exist*"]
[Escape Map
  
]
юникод
shit: 1:14: Error: Could not expand path: No expansions found for 'does_not_exist*'.
     1 |  echo юникод; does_not_exist*
       |               ^~~~~~~~~~~~~~~ here.
